README.md

```diff
@@ -1,63 +1,67 @@
-# Tascurator-frontend
+<div align="center">
+	<img src="public/logo.svg" alt="Tascurator Logo" width="120" />
+</div>
 
-## Getting Started
+<br>
 
-First, install the dependencies:
+**Tascurator** is a web application designed to streamline the management of cleaning duties in a share house.
+It offers a user-friendly platform for landlords to effortlessly create tasks with detailed instructions, invite tenants, and track progress.
+Tenants can quickly access, update, and report on their tasks, ensuring a well-organized living environment.
 
-```bash
-npm install
-```
+## Key Features
 
-Second, run Docker container:
+### For landlords:
 
-```bash
-docker compose up
-```
+- **Task Management**: Create and assign tasks to tenants with ease.
```

Comment:
In my opinion, it's better to include "landlord can group multiple tasks together"